web/src/i18n/locales/en/proposedBudget.yaml

```diff
@@ -23,8 +32,19 @@ combinedActivityCosts:
       Computable Medicaid Cost** by subtracting other costs that  were described
       in each activity. The other costs provided here are the sum of other costs
       from each activity.
+  activityBreakdownMMIS:
+    title: State and Contractor Cost Breakdown
+    detail: >-
+      These tables summarize **Key State Personnel**, **State Staff**, **Other
+      State Expenses**, and **Private Contractors** for all activities in this
+      APD, organized by FFY. These tables provide subtotals for each FFY and
+      calculate the **Total Computable Medicaid Cost** by subtracting other
+      costs that were described in each activity. The other costs provided here
+      are the sum of other costs from each activity.
 summaryBudget:
   title: Summary Budget Table
```

Comment:
**(Non-blocking)** It seems like you might be able to get rid of this since it's replaced by `summaryBudgetHITECH` & `summaryBudgetMMIS`

Reply:
I thought so too but it's still used by the navigation. I thought about switching the nav to use either `summaryBudgetHITECH` or `summaryBudgetMMIS` but decided to just have it redundant until we drop HITECH.